.github/workflows/phoenix/test.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+build_opts=""
+if [ "$job_device" == "gpu" ]; then
+    build_opts="--gpu"
+fi
+
+./mfc.sh build -j $(nproc) $build_opts
+
+n_test_threads=$(nproc)
+
+if [ "$job_device" == "gpu" ]; then
+    gpu_count=$(nvidia-smi -L | wc -l)        # number of GPUs on node
+    gpu_ids=$(seq -s ' ' 0 $(($gpu_count-1))) # 0,1,2,...,gpu_count-1
+    device_opts="-g $gpu_ids"
+    n_test_threads=`expr $gpu_count \* 2`
+fi
+
+./mfc.sh test -a -j $n_test_threads $device_opts -- -b mpirun -f --bind-to none
```

Comment:
FYI be careful with this, the current `run-phoenix-release-gpu.sh` file prescribes $(nproc) threads to `./mfc.sh test`, which oversubscribes all of the resources (we only request 2 GPUs). This is causing it to spuriously fail.

Reply:
Indeed, I noticed this and had changed it to:
```bash
n_test_threads=`expr $gpu_count \* 2`
```
in GPU mode.